hoodie-client/src/main/java/com/uber/hoodie/CompactionAdminClient.java

```diff
@@ -65,7 +65,7 @@
   private static Logger log = LogManager.getLogger(CompactionAdminClient.class);
 
   public CompactionAdminClient(JavaSparkContext jsc, String basePath) {
-    super(jsc, HoodieWriteConfig.newBuilder().withPath(basePath).build());
+    super(jsc, HoodieWriteConfig.newBuilder().withPath(basePath).build(), java.util.Optional.empty());
```

Comment:
make this into another overload constructor in AbstractHoodieClient itself?

Reply:
otherwise, every piece of code needs to remember the special meaning of Optional.empty arg.

Reply:
Done